CMakeLists.txt

```diff
@@ -5,34 +5,21 @@
  #              GNU Lesser General Public Licence (LGPL) version 3,             #
  #                  copied verbatim in the file "LICENSE"                       #
  ################################################################################
-# The name of our project is "FairBase".  CMakeLists files in this project can
+# The name of our project is "FairRoot".  CMakeLists files in this project can
 # refer to the root source directory of the project as ${FAIRROOT_SOURCE_DIR} or
 # as ${CMAKE_SOURCE_DIR} and to the root binary directory of the project as
 # ${FAIRROOT_BINARY_DIR} or ${CMAKE_BINARY_DIR}.
 # This difference is important for the base classes which are in FAIRROOT
-# and PANDAROOT.
+# and the subproject (i.e: CbmRoot, Pandaroot,... etc).
 
 # Check if cmake has the required version
 CMAKE_MINIMUM_REQUIRED(VERSION 3.11.0 FATAL_ERROR)
 
-### CMP0025   Compiler id for Apple Clang is now AppleClang.
-### CMP0042   MACOSX_RPATH is enabled by default.
-
-foreach(p
-  CMP0025 # CMake 3.0
-  CMP0028 # double colon for imported and alias targets
-  CMP0042 # CMake 3.0
-  CMP0054 # Only interpret ``if()`` arguments as variables or keywords when unquoted.
-  )
-  if(POLICY ${p})
-  cmake_policy(SET ${p} NEW)
-  endif()
-endforeach()
-
+set(CMAKE_INSTALL_MESSAGE LAZY)
 # Set project version
 SET(FAIRROOT_MAJOR_VERSION 18)
 SET(FAIRROOT_MINOR_VERSION 0)
-SET(FAIRROOT_PATCH_VERSION 1)
+SET(FAIRROOT_PATCH_VERSION 3)
```

Comment:
We are already at patch version 6 for the v18.0 branch. So for the development branch we should choose a higher number or decide on a different numbering scheme.

Reply:
Ok, I will change it to 7 now.